src/App.css

```diff
@@ -108,28 +52,6 @@
   font-size: 20px;
 }
 
-.navbar{
-  position: fixed;
-  border-bottom-right-radius: 1em;
-  border-bottom:solid;
-  width:100%;
-  background-color: #495056;
-}
-
-.navbar-brand{
```

Comment:
Same question here as for the above. Make sure that any change you make in a pull request is exactly doing what the ticket requires. Otherwise if some of this is simply clean up work, you may need to create a separate ticket and hence pull request, for it.

Reply:
I moved all the css that was affecting the login, and Navbar to a new css file called index.css so as to maintain the best practices. That's why it shows those deletes